src/foam/blob/Blob.js

```diff
@@ -175,12 +202,15 @@ foam.CLASS({
   extends: 'foam.blob.AbstractBlob',
   properties: [
     {
+      class: 'foam.core.Blob',
```

Comment:
Just clasS: 'Blob' is fine.  foam.core is the default

Reply:
Removed the prefix